### ISSUE TITLE
[Bug]: Todays Alpha: MessageBox Text does not respect Carriage returns in Text

### ISSUE THREAD

**Issue by Smurf-IV** [Krypton-Suite/Standard-Toolkit#1660]
![image](https://github.com/user-attachments/assets/84d7e9b4-6f83-405e-b146-441ec376a68a)

**Comment by Wagnerp** [comment]
This may help https://stackoverflow.com/questions/15433188/what-is-the-difference-between-r-n-r-and-n

**Comment by Ahmed-Abdelhameed** [comment]
This is a side effect of using a KTextBox in KMessageBox. Should we try and fix this in the source (i.e., KTextBox) or only in KMessageBox?

I did some testing and here's what I found:

- `Label` supports `\r`, `\n`, and `\r\n`.
- `TextBox` supports `\n`, `\r\n`, but _not_ `\r`.
- `KryptonLabel` supports `\n`, `\r\n`, but _not_ `\r`.
- `KryptonTextBox` only supports `\r\n`.
- `KryptonWrapLabel` works exactly like the WinForms `Label` (which is expected since it's derived from it and lets it handle the painting).
